purchase_order_sale_order_nrq/data/ir_action_data.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="purchase_order_sale_ids_initialize" model="ir.actions.server">
```

Comment:
I suppose we do not need these records for updating stored computed fields.  Let me know this is not the case.

Reply:
Yes, you are correct. It behaves strangely on my development environment that the invoice side's fields are not initialized. Let me remove this part for now